### PR TITLE
Added new sounds for asteroid explosions

### PR DESCRIPTION
**Pull request by Floppa-Priest**
## Summary

I added three new sounds for small, medium and large asteroid explosions.

harvesting.txt, effects.txt and copyright have all been updated as well.
## PR Checklist
 - [x] I updated the copyright attributions, or decline to claim copyright of any assets produced or modified
 - [ ] I uploaded the necessary image, blend, and texture assets here: {{insert link to assets}}
 - [ ] I created a PR to the [endless-sky-high-dpi repo](https://github.com/endless-sky/endless-sky-high-dpi) with the `@2x` versions of these art assets: {{insert PR link}}

### CODE REVIEW

**Comment by Floppa-Priest** [comment]
Before this gets approved, can someone check to make sure I got my copyright additions correct?

File: data/effects.txt [hunk 1]
```diff
@@ -168,7 +168,14 @@ effect "final explosion medium"
...
 	sound "final explosion large"
 
+effect "asteroid crunch small"
+	sound "asteroid crunch small"
 
+effect "asteroid crunch medium"
+	sound "asteroid crunch medium"
+	
+effect "asteroid crunch large"
+	sound "asteroid crunch large"
```
Comment (Amazinite) [inline data/effects.txt:178]:
```suggestion
	sound "asteroid crunch large"
	
```

**Review by Amazinite — CHANGES_REQUESTED** [review]
The copyright looks good to me.

File: data/effects.txt (hunk 1 above)
Comment (Floppa-Priest) [reply data/effects.txt]: Good catch

**Review by Amazinite — APPROVED** [review]

**Comment by Amazinite** [comment]
Sounds good to me.

**Pull request merged**